src/main/webapp/view/transcript/transcript.js

```diff
@@ -149,16 +150,22 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
 
-/** Returns true if`transcriptLineLiElement` is bolded. */
+/** Returns true if `transcriptLineLiElement` is bolded. */
 function isBolded(transcriptLineLiElement) {
   return transcriptLineLiElement.classList.contains(BOLD_FONT_WEIGHT);
 }
 
 /**
  * Returns true if `currentTimeMs` is within the time range for
- * the current transcript line.
+ * 'transcriptLine'
+ *
+ * <p>If `transcriptLine` is undefined, `currentTimeMs` is checked to be
+ * within the current transcript line instead.
  */
-function isWithinCurrentTimeRange(currentTimeMs) {
+function isWithinCurrentTimeRange(currentTimeMs, transcriptLine) {
```

Comment:
Technically this doesnt need to be checking the currentTime, can you rename the function to something like `isTimeWithinTranscriptLine(timeMs, transcriptLine)`?